packages/infra/lib/location-services-stack.ts

```diff
@@ -13,6 +13,9 @@ export class LocationServicesStack extends Stack {
     //-------------------------------------------
     // API Gateway
     //-------------------------------------------
+    if (!props.config.locationService) {
+      return;
+    }
```

Comment:
Can't we have this on `infrastructure.ts` so the stack is not even created on sandbox?

Reply:
I already have it on `infrastructure.ts`.
In this file, `props.config.locationService` might be undefined, so I added this condition here as well.